feed.xml

```diff
@@ -12,12 +12,14 @@ layout: null
     <pubDate>{{ site.time | date_to_rfc822 }}</pubDate>
     <lastBuildDate>{{ site.time | date_to_rfc822 }}</lastBuildDate>
     <generator>Jekyll v{{ jekyll.version }}</generator>
-    {% for page in site.html_pages %}
+    {% assign sorted_pages = site.html_pages | sort:"last_modified_at" | reverse %}
+    {% for page in sorted_pages %}
       {% unless page.tag == 'omit' %}
```

Comment:
if  we suppose to use the tag, it should be moved to assignment filters, for example:
```
{% assign sorted_pages = site.html_pages | where_exp: "page", "page.tag != 'omit'"| sort:"last_modified_at" | reverse %}
```
If we don't have real cases to use it, it's better to remove it for now to not overload the generation logic. We can add it when we'll have the corresponding cases.

Reply:
I'm ok with that. I just wanted to make sure that we could ignore certain files if we needed to. We're not currently using this on devdocs. 

It may be necessary for the OMS docs site, however. For example, to exclude the German translation of pages based on the `lang:` metadata tag so that we build feeds for each language appropriately. Otherwise, the feed will include both languages, which is not good.